src/main/java/org/ja13/eau/EAU.java

```diff
@@ -135,7 +135,7 @@
     public final static String URL = "https://eau.ja13.org";
     public final static String UPDATE_URL = "https://github.com/jrddunbr/EAU/releases";
     public final static String SRC_URL = "https://github.com/jrddunbr/EAU";
-    public final static String[] AUTHORS = {"jrddunbr"};
+    public final static String[] AUTHORS = {"jrddunbr,Omega Haxors"};
```

Comment:
You and I are not the same person. This is a list of strings. Get your own darn string! :D

```java
public final static String[] AUTHORS = {"jrddunbr", "Omega Haxors"};
```

Reply:
Bunk Beds!